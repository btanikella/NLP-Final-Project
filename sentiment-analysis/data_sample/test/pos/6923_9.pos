DT NN VBZ DT NN TO VB CC MD VB RB IN NN CC NN
PRP VBP PRP RB VBP TO VB JJ TO VB DT DT IN DT NNS JJ IN NN  NNS CC DT NN NN IN DT IN DT NNS CC VB PRP VB PRP IN WRB NNP NNP VBZ  NNP  DT NN VBZ RB JJ TO JJS NNS IN NNP IN PRP VBP JJ NNS IN PRP$ JJ NN WDT VBP  VBP CC VBP IN DT
DT JJ JJ NN VBN CC NNP NNP NN VBG RB VBP JJ DT NN CC IN NN IN DT DT NN VBZ JJ  DT NN VBZ JJ CC DT NN RB VBG CC NN
DT JJ NN NN NN IN DT JJ NN IN NNP NNP
