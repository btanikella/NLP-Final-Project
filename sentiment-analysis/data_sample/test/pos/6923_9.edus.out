Sentence #1 (16 tokens):
This film is a joy to watch and should do well on DVD and video.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=9 CharacterOffsetEnd=13 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=14 CharacterOffsetEnd=16 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=17 CharacterOffsetEnd=18 PartOfSpeech=DT] [Text=joy CharacterOffsetBegin=19 CharacterOffsetEnd=22 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=23 CharacterOffsetEnd=25 PartOfSpeech=TO] [Text=watch CharacterOffsetBegin=26 CharacterOffsetEnd=31 PartOfSpeech=VB] [Text=and CharacterOffsetBegin=32 CharacterOffsetEnd=35 PartOfSpeech=CC] [Text=should CharacterOffsetBegin=36 CharacterOffsetEnd=42 PartOfSpeech=MD] [Text=do CharacterOffsetBegin=43 CharacterOffsetEnd=45 PartOfSpeech=VB] [Text=well CharacterOffsetBegin=46 CharacterOffsetEnd=50 PartOfSpeech=RB] [Text=on CharacterOffsetBegin=51 CharacterOffsetEnd=53 PartOfSpeech=IN] [Text=DVD CharacterOffsetBegin=54 CharacterOffsetEnd=57 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=58 CharacterOffsetEnd=61 PartOfSpeech=CC] [Text=video CharacterOffsetBegin=62 CharacterOffsetEnd=67 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=67 CharacterOffsetEnd=68 PartOfSpeech=.] 
root(ROOT-0, joy-5)
det(film-2, This-1)
nsubj(joy-5, film-2)
cop(joy-5, is-3)
det(joy-5, a-4)
aux(watch-7, to-6)
infmod(joy-5, watch-7)
cc(watch-7, and-8)
aux(do-10, should-9)
conj(watch-7, do-10)
advmod(do-10, well-11)
prep(do-10, on-12)
pobj(on-12, DVD-13)
cc(DVD-13, and-14)
conj(DVD-13, video-15)
punct(joy-5, .-16)

Sentence #2 (70 tokens):
I suppose you really have to be Irish to appreciate the some of the subtlties such as accent, colloquialisms and the dress sense of some of the characters but let me assure you that when Dylan Moran impersonates 'Barreler' the impersonation is quite familiar to most people from Dublin because we have many characters in our fine city that look, act and talk like that!
[Text=I CharacterOffsetBegin=73 CharacterOffsetEnd=74 PartOfSpeech=PRP] [Text=suppose CharacterOffsetBegin=75 CharacterOffsetEnd=82 PartOfSpeech=VBP] [Text=you CharacterOffsetBegin=83 CharacterOffsetEnd=86 PartOfSpeech=PRP] [Text=really CharacterOffsetBegin=87 CharacterOffsetEnd=93 PartOfSpeech=RB] [Text=have CharacterOffsetBegin=94 CharacterOffsetEnd=98 PartOfSpeech=VBP] [Text=to CharacterOffsetBegin=99 CharacterOffsetEnd=101 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=102 CharacterOffsetEnd=104 PartOfSpeech=VB] [Text=Irish CharacterOffsetBegin=105 CharacterOffsetEnd=110 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=111 CharacterOffsetEnd=113 PartOfSpeech=TO] [Text=appreciate CharacterOffsetBegin=114 CharacterOffsetEnd=124 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=125 CharacterOffsetEnd=128 PartOfSpeech=DT] [Text=some CharacterOffsetBegin=129 CharacterOffsetEnd=133 PartOfSpeech=DT] [Text=of CharacterOffsetBegin=134 CharacterOffsetEnd=136 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=137 CharacterOffsetEnd=140 PartOfSpeech=DT] [Text=subtlties CharacterOffsetBegin=141 CharacterOffsetEnd=150 PartOfSpeech=NNS] [Text=such CharacterOffsetBegin=151 CharacterOffsetEnd=155 PartOfSpeech=JJ] [Text=as CharacterOffsetBegin=156 CharacterOffsetEnd=158 PartOfSpeech=IN] [Text=accent CharacterOffsetBegin=159 CharacterOffsetEnd=165 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=165 CharacterOffsetEnd=166 PartOfSpeech=,] [Text=colloquialisms CharacterOffsetBegin=167 CharacterOffsetEnd=181 PartOfSpeech=NNS] [Text=and CharacterOffsetBegin=182 CharacterOffsetEnd=185 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=186 CharacterOffsetEnd=189 PartOfSpeech=DT] [Text=dress CharacterOffsetBegin=190 CharacterOffsetEnd=195 PartOfSpeech=NN] [Text=sense CharacterOffsetBegin=196 CharacterOffsetEnd=201 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=202 CharacterOffsetEnd=204 PartOfSpeech=IN] [Text=some CharacterOffsetBegin=205 CharacterOffsetEnd=209 PartOfSpeech=DT] [Text=of CharacterOffsetBegin=210 CharacterOffsetEnd=212 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=213 CharacterOffsetEnd=216 PartOfSpeech=DT] [Text=characters CharacterOffsetBegin=217 CharacterOffsetEnd=227 PartOfSpeech=NNS] [Text=but CharacterOffsetBegin=228 CharacterOffsetEnd=231 PartOfSpeech=CC] [Text=let CharacterOffsetBegin=232 CharacterOffsetEnd=235 PartOfSpeech=VB] [Text=me CharacterOffsetBegin=236 CharacterOffsetEnd=238 PartOfSpeech=PRP] [Text=assure CharacterOffsetBegin=239 CharacterOffsetEnd=245 PartOfSpeech=VB] [Text=you CharacterOffsetBegin=246 CharacterOffsetEnd=249 PartOfSpeech=PRP] [Text=that CharacterOffsetBegin=250 CharacterOffsetEnd=254 PartOfSpeech=IN] [Text=when CharacterOffsetBegin=255 CharacterOffsetEnd=259 PartOfSpeech=WRB] [Text=Dylan CharacterOffsetBegin=260 CharacterOffsetEnd=265 PartOfSpeech=NNP] [Text=Moran CharacterOffsetBegin=266 CharacterOffsetEnd=271 PartOfSpeech=NNP] [Text=impersonates CharacterOffsetBegin=272 CharacterOffsetEnd=284 PartOfSpeech=VBZ] [Text=` CharacterOffsetBegin=285 CharacterOffsetEnd=286 PartOfSpeech=``] [Text=Barreler CharacterOffsetBegin=286 CharacterOffsetEnd=294 PartOfSpeech=NNP] [Text=' CharacterOffsetBegin=294 CharacterOffsetEnd=295 PartOfSpeech=''] [Text=the CharacterOffsetBegin=296 CharacterOffsetEnd=299 PartOfSpeech=DT] [Text=impersonation CharacterOffsetBegin=300 CharacterOffsetEnd=313 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=314 CharacterOffsetEnd=316 PartOfSpeech=VBZ] [Text=quite CharacterOffsetBegin=317 CharacterOffsetEnd=322 PartOfSpeech=RB] [Text=familiar CharacterOffsetBegin=323 CharacterOffsetEnd=331 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=332 CharacterOffsetEnd=334 PartOfSpeech=TO] [Text=most CharacterOffsetBegin=335 CharacterOffsetEnd=339 PartOfSpeech=JJS] [Text=people CharacterOffsetBegin=340 CharacterOffsetEnd=346 PartOfSpeech=NNS] [Text=from CharacterOffsetBegin=347 CharacterOffsetEnd=351 PartOfSpeech=IN] [Text=Dublin CharacterOffsetBegin=352 CharacterOffsetEnd=358 PartOfSpeech=NNP] [Text=because CharacterOffsetBegin=359 CharacterOffsetEnd=366 PartOfSpeech=IN] [Text=we CharacterOffsetBegin=367 CharacterOffsetEnd=369 PartOfSpeech=PRP] [Text=have CharacterOffsetBegin=370 CharacterOffsetEnd=374 PartOfSpeech=VBP] [Text=many CharacterOffsetBegin=375 CharacterOffsetEnd=379 PartOfSpeech=JJ] [Text=characters CharacterOffsetBegin=380 CharacterOffsetEnd=390 PartOfSpeech=NNS] [Text=in CharacterOffsetBegin=391 CharacterOffsetEnd=393 PartOfSpeech=IN] [Text=our CharacterOffsetBegin=394 CharacterOffsetEnd=397 PartOfSpeech=PRP$] [Text=fine CharacterOffsetBegin=398 CharacterOffsetEnd=402 PartOfSpeech=JJ] [Text=city CharacterOffsetBegin=403 CharacterOffsetEnd=407 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=408 CharacterOffsetEnd=412 PartOfSpeech=WDT] [Text=look CharacterOffsetBegin=413 CharacterOffsetEnd=417 PartOfSpeech=VBP] [Text=, CharacterOffsetBegin=417 CharacterOffsetEnd=418 PartOfSpeech=,] [Text=act CharacterOffsetBegin=419 CharacterOffsetEnd=422 PartOfSpeech=VBP] [Text=and CharacterOffsetBegin=423 CharacterOffsetEnd=426 PartOfSpeech=CC] [Text=talk CharacterOffsetBegin=427 CharacterOffsetEnd=431 PartOfSpeech=VBP] [Text=like CharacterOffsetBegin=432 CharacterOffsetEnd=436 PartOfSpeech=IN] [Text=that CharacterOffsetBegin=437 CharacterOffsetEnd=441 PartOfSpeech=DT] [Text=! CharacterOffsetBegin=441 CharacterOffsetEnd=442 PartOfSpeech=.] 
root(ROOT-0, suppose-2)
nsubj(suppose-2, I-1)
nsubj(have-5, you-3)
advmod(have-5, really-4)
ccomp(suppose-2, have-5)
aux(Irish-8, to-6)
cop(Irish-8, be-7)
xcomp(have-5, Irish-8)
aux(appreciate-10, to-9)
dep(Irish-8, appreciate-10)
det(some-12, the-11)
dobj(appreciate-10, some-12)
prep(some-12, of-13)
det(subtlties-15, the-14)
pobj(of-13, subtlties-15)
mwe(as-17, such-16)
prep(subtlties-15, as-17)
pobj(as-17, accent-18)
punct(accent-18, ,-19)
conj(accent-18, colloquialisms-20)
cc(accent-18, and-21)
det(sense-24, the-22)
nn(sense-24, dress-23)
conj(accent-18, sense-24)
prep(sense-24, of-25)
pobj(of-25, some-26)
prep(some-26, of-27)
det(characters-29, the-28)
pobj(of-27, characters-29)
cc(appreciate-10, but-30)
conj(appreciate-10, let-31)
nsubj(assure-33, me-32)
ccomp(let-31, assure-33)
dobj(assure-33, you-34)
dep(assure-33, that-35)
advmod(impersonates-39, when-36)
nn(Moran-38, Dylan-37)
nsubj(impersonates-39, Moran-38)
dep(that-35, impersonates-39)
punct(Barreler-41, `-40)
dep(familiar-47, Barreler-41)
punct(Barreler-41, '-42)
det(impersonation-44, the-43)
nsubj(familiar-47, impersonation-44)
cop(familiar-47, is-45)
advmod(familiar-47, quite-46)
dobj(impersonates-39, familiar-47)
prep(familiar-47, to-48)
amod(people-50, most-49)
pobj(to-48, people-50)
prep(familiar-47, from-51)
pobj(from-51, Dublin-52)
mark(have-55, because-53)
nsubj(have-55, we-54)
advcl(impersonates-39, have-55)
amod(characters-57, many-56)
dobj(have-55, characters-57)
prep(characters-57, in-58)
poss(city-61, our-59)
amod(city-61, fine-60)
pobj(in-58, city-61)
nsubj(look-63, that-62)
rcmod(characters-57, look-63)
punct(impersonates-39, ,-64)
conj(impersonates-39, act-65)
cc(impersonates-39, and-66)
conj(impersonates-39, talk-67)
prep(talk-67, like-68)
pobj(like-68, that-69)
punct(suppose-2, !-70)

Sentence #3 (37 tokens):
The sheer simple comedy employed and Michael Caines genius acting alone are worth the money but on top of this the plot is great, the script is fantastic and the dialogue fast moving and catchy.
[Text=The CharacterOffsetBegin=447 CharacterOffsetEnd=450 PartOfSpeech=DT] [Text=sheer CharacterOffsetBegin=451 CharacterOffsetEnd=456 PartOfSpeech=JJ] [Text=simple CharacterOffsetBegin=457 CharacterOffsetEnd=463 PartOfSpeech=JJ] [Text=comedy CharacterOffsetBegin=464 CharacterOffsetEnd=470 PartOfSpeech=NN] [Text=employed CharacterOffsetBegin=471 CharacterOffsetEnd=479 PartOfSpeech=VBN] [Text=and CharacterOffsetBegin=480 CharacterOffsetEnd=483 PartOfSpeech=CC] [Text=Michael CharacterOffsetBegin=484 CharacterOffsetEnd=491 PartOfSpeech=NNP] [Text=Caines CharacterOffsetBegin=492 CharacterOffsetEnd=498 PartOfSpeech=NNP] [Text=genius CharacterOffsetBegin=499 CharacterOffsetEnd=505 PartOfSpeech=NN] [Text=acting CharacterOffsetBegin=506 CharacterOffsetEnd=512 PartOfSpeech=VBG] [Text=alone CharacterOffsetBegin=513 CharacterOffsetEnd=518 PartOfSpeech=RB] [Text=are CharacterOffsetBegin=519 CharacterOffsetEnd=522 PartOfSpeech=VBP] [Text=worth CharacterOffsetBegin=523 CharacterOffsetEnd=528 PartOfSpeech=JJ] [Text=the CharacterOffsetBegin=529 CharacterOffsetEnd=532 PartOfSpeech=DT] [Text=money CharacterOffsetBegin=533 CharacterOffsetEnd=538 PartOfSpeech=NN] [Text=but CharacterOffsetBegin=539 CharacterOffsetEnd=542 PartOfSpeech=CC] [Text=on CharacterOffsetBegin=543 CharacterOffsetEnd=545 PartOfSpeech=IN] [Text=top CharacterOffsetBegin=546 CharacterOffsetEnd=549 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=550 CharacterOffsetEnd=552 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=553 CharacterOffsetEnd=557 PartOfSpeech=DT] [Text=the CharacterOffsetBegin=558 CharacterOffsetEnd=561 PartOfSpeech=DT] [Text=plot CharacterOffsetBegin=562 CharacterOffsetEnd=566 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=567 CharacterOffsetEnd=569 PartOfSpeech=VBZ] [Text=great CharacterOffsetBegin=570 CharacterOffsetEnd=575 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=575 CharacterOffsetEnd=576 PartOfSpeech=,] [Text=the CharacterOffsetBegin=577 CharacterOffsetEnd=580 PartOfSpeech=DT] [Text=script CharacterOffsetBegin=581 CharacterOffsetEnd=587 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=588 CharacterOffsetEnd=590 PartOfSpeech=VBZ] [Text=fantastic CharacterOffsetBegin=591 CharacterOffsetEnd=600 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=601 CharacterOffsetEnd=604 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=605 CharacterOffsetEnd=608 PartOfSpeech=DT] [Text=dialogue CharacterOffsetBegin=609 CharacterOffsetEnd=617 PartOfSpeech=NN] [Text=fast CharacterOffsetBegin=618 CharacterOffsetEnd=622 PartOfSpeech=RB] [Text=moving CharacterOffsetBegin=623 CharacterOffsetEnd=629 PartOfSpeech=VBG] [Text=and CharacterOffsetBegin=630 CharacterOffsetEnd=633 PartOfSpeech=CC] [Text=catchy CharacterOffsetBegin=634 CharacterOffsetEnd=640 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=640 CharacterOffsetEnd=641 PartOfSpeech=.] 
root(ROOT-0, worth-13)
det(comedy-4, The-1)
amod(comedy-4, sheer-2)
amod(comedy-4, simple-3)
nsubj(worth-13, comedy-4)
partmod(comedy-4, employed-5)
cc(employed-5, and-6)
nn(genius-9, Michael-7)
nn(genius-9, Caines-8)
conj(employed-5, genius-9)
dep(employed-5, acting-10)
advmod(acting-10, alone-11)
cop(worth-13, are-12)
det(money-15, the-14)
dep(worth-13, money-15)
cc(money-15, but-16)
conj(money-15, on-17)
pobj(on-17, top-18)
prep(top-18, of-19)
pobj(of-19, this-20)
det(plot-22, the-21)
nsubj(great-24, plot-22)
cop(great-24, is-23)
rcmod(this-20, great-24)
punct(fantastic-29, ,-25)
det(script-27, the-26)
nsubj(fantastic-29, script-27)
cop(fantastic-29, is-28)
dep(great-24, fantastic-29)
cc(fantastic-29, and-30)
det(dialogue-32, the-31)
conj(fantastic-29, dialogue-32)
advmod(moving-34, fast-33)
partmod(dialogue-32, moving-34)
cc(moving-34, and-35)
conj(moving-34, catchy-36)
punct(worth-13, .-37)

Sentence #4 (13 tokens):
A perfect light entertainment movie without the madcap humour of Jim Carrey.
[Text=A CharacterOffsetBegin=646 CharacterOffsetEnd=647 PartOfSpeech=DT] [Text=perfect CharacterOffsetBegin=648 CharacterOffsetEnd=655 PartOfSpeech=JJ] [Text=light CharacterOffsetBegin=656 CharacterOffsetEnd=661 PartOfSpeech=NN] [Text=entertainment CharacterOffsetBegin=662 CharacterOffsetEnd=675 PartOfSpeech=NN] [Text=movie CharacterOffsetBegin=676 CharacterOffsetEnd=681 PartOfSpeech=NN] [Text=without CharacterOffsetBegin=682 CharacterOffsetEnd=689 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=690 CharacterOffsetEnd=693 PartOfSpeech=DT] [Text=madcap CharacterOffsetBegin=694 CharacterOffsetEnd=700 PartOfSpeech=JJ] [Text=humour CharacterOffsetBegin=701 CharacterOffsetEnd=707 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=708 CharacterOffsetEnd=710 PartOfSpeech=IN] [Text=Jim CharacterOffsetBegin=711 CharacterOffsetEnd=714 PartOfSpeech=NNP] [Text=Carrey CharacterOffsetBegin=715 CharacterOffsetEnd=721 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=721 CharacterOffsetEnd=722 PartOfSpeech=.] 
root(ROOT-0, movie-5)
det(movie-5, A-1)
amod(movie-5, perfect-2)
nn(movie-5, light-3)
nn(movie-5, entertainment-4)
prep(movie-5, without-6)
det(humour-9, the-7)
amod(humour-9, madcap-8)
pobj(without-6, humour-9)
prep(humour-9, of-10)
nn(Carrey-12, Jim-11)
pobj(of-10, Carrey-12)
punct(movie-5, .-13)

